components/bl-audio-recorder-component/src/index.js

```diff
@@ -111,7 +121,16 @@ export default function AudioRecorder({ component, eventHandlers, elRef }) {
           <button
             disabled={ state && state !== StreamState.INACTIVE }
             className="control-button" onClick={ startRecording }>
-            { startText }
+            { state && state !== StreamState.INACTIVE ?
+              (
+                <>
+                  <span className="record-dot"/>
+                  <span className="record-text">rec</span>
```

Comment:
![image](https://github.com/Backendless/ui-builder-library/assets/15347646/e7fbb80a-ca7c-4f5b-a5a6-e52337948563)

I'm not sure, but looks like it not completely symmetric. Could you make a precise look into it?